crates/storage-api/src/timer_table/mod.rs

```diff
@@ -138,15 +158,15 @@ impl restate_types::timer::TimerKey for TimerKey {
 #[derive(Clone, Debug, Eq, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub enum Timer {
-    CompleteSleepEntry(InvocationId, u32),
     Invoke(ServiceInvocation),
+    CompleteJournalEntry(InvocationId, u32),
     CleanInvocationStatus(InvocationId),
 }
 
 impl WithPartitionKey for Timer {
```

Comment:
Side note: at some point i think it would be useful to have a trait similar to this for invocation id: `WithInvocationId`.